### PR TITLE
[fix][KLD-185]: message about not finding data

### PR DESCRIPTION
**Pull request by RDMStreet**

### CODE REVIEW

File: src/modules/ModuleFarming/index.vue [hunk 1]
```diff
@@ -124,15 +124,26 @@ const expandPools = computed(() => poolsPaginated.value?.length === 1)
...
+      v-if="!poolsFinal?.length && !isLoading"
+      :class="$style.empty"
       class="flex-1 flex items-center justify-center"
     >
-      There are no farming pools at the moment
+      {{
+        !farming?.pools.length
+          ? 'There are no farming pools at the moment'
+          : 'There are no farming pools match the current filter'
+      }}
```
Comment (0x009922) [inline src/modules/ModuleFarming/index.vue:140]:
```suggestion
      <template v-if="farming?.pools.length">
        There are no farming pools match the current filter
      </template>
      <template v-else>
        There are no farming pools at the moment
      </template>
```

File: src/modules/ModuleStaking/index.vue [hunk 2]
```diff
@@ -111,23 +109,34 @@ const poolsPaginated = computed<Pool[] | null>(() => {
 
 const poolsFinal = poolsPaginated
 
-const isLoading = PoolsQuery.loading
+const isLoading = computed(() => poolsFinal.value === null)
```
Comment (0x009922) [inline src/modules/ModuleStaking/index.vue:112]:
```suggestion
const isLoading = computed(() => !poolsFinal.value)
```

File: src/modules/ModuleStaking/index.vue [hunk 3]
```diff
@@ -111,23 +109,34 @@ const poolsPaginated = computed<Pool[] | null>(() => {
...
+      v-if="!poolsFinal?.length && !isLoading"
+      :class="$style.empty"
       class="flex-1 flex items-center justify-center"
     >
-      There are no staking pools at the moment
+      {{
+        !pools?.length
+          ? 'There are no staking pools at the moment'
+          : 'There are no staking pools match the current filter'
+      }}
```
Comment (0x009922) [inline src/modules/ModuleStaking/index.vue:133]: Same as in `ModuleFarming/index.vue` - with `<template v-if>` it looks easier (imho).

File: src/modules/ModuleStaking/index.vue [hunk 4]
```diff
@@ -154,32 +163,33 @@ function handleStakedUnstaked() {
...
+
+.loader {
+  min-height: 82px + $padding-bottom;
+  margin-bottom: -$padding-bottom;
+}
+
+.empty {
+  color: vars.$gray3;
+}
 </style>
```
Comment (0x009922) [inline src/modules/ModuleStaking/index.vue:195]: Looks like you're repeating yourself. Maybe, move that CSS module into a separate file?

**Review by 0x009922 — APPROVED** [review]

File: src/modules/ModuleStaking/index.vue (hunk 3 above)
Comment (RDMStreet) [reply src/modules/ModuleStaking/index.vue:133]: Changed

File: src/modules/ModuleStaking/index.vue (hunk 4 above)
Comment (RDMStreet) [reply src/modules/ModuleStaking/index.vue:195]: Yes, I agree. I've been thinking about it. I think to put it in a separate component. I'll do it in another PR

**Pull request merged**